extensions/kubectl-cli/src/extension.ts

```diff
@@ -362,6 +363,26 @@ async function postActivate(
       releaseVersionToInstall = undefined;
       releaseToInstall = undefined;
     },
+    doUninstall: async _logger => {
+      if (!currentVersion) {
+        throw new Error(`Cannot uninstall ${kubectlCliName}. No version detected.`);
+      }
+
+      // delete the executable stored in the storage folder
+      const storagePath = getStorageKubectlPath(extensionContext);
+      if (fs.existsSync(storagePath)) {
+        await fs.promises.unlink(storagePath);
+      }
+
+      // delete the executable in the system path
+      const systemPath = getSystemBinaryPath(kubectlCliName);
+      if (fs.existsSync(systemPath)) {
+        await fs.promises.unlink(systemPath);
```

Comment:
We need admin privilege to remove on the system path

![image](https://github.com/user-attachments/assets/f99eefe4-67c7-4568-b414-6e86faa2982e)

Reply:
Thanks for noticing, i was doing a test with the admin user and i didn't face it